pkg/ddl/executor.go

```diff
@@ -4576,7 +4535,7 @@ func (e *executor) CreatePrimaryKey(ctx sessionctx.Context, ti ast.Ident, indexN
 		Type:           model.ActionAddPrimaryKey,
 		BinlogInfo:     &model.HistoryInfo{},
 		ReorgMeta:      nil,
-		Args:           []any{unique, indexName, indexPartSpecifications, indexOption, sqlMode, nil, global},
+		Args:           []any{unique, indexName, indexPartSpecifications, indexOption, sqlMode, nil},
```

Comment:
Will deleting this affect the upgrade? /cc @tangenta

Reply:
No, removing arguments from job.Args is handled gracefully (simply ignored and DecodeArgs will regenerate RawArgs IIRC. So the result should be that old queued DDLs should fail due to lack of explicitly set GLOBAL (since it did not have IndexOption set).

Reply:
I think we should leave a placeholder here to distinguish it from new argument introduced in the future. Otherwise, there may be an unmarshal error during upgrade.

Reply:
Ah, right if it is queued with a newer version and unqueued by an old version there might be issues, I will add a placeholder.